catalog/core/catalog-core-commons/src/test/java/ddf/catalog/impl/filter/GeoToolsFunctionFactoryTest.java

```diff
@@ -25,48 +27,57 @@
 import org.opengis.filter.expression.Expression;
 import org.opengis.filter.expression.Function;
 
-public class FuzzyFunctionFactoryTest {
-    FuzzyFunctionFactory toTest;
+public class GeoToolsFunctionFactoryTest {
+    GeoToolsFunctionFactory toTest;
 
     @Before
     public void setup() {
-        toTest = new FuzzyFunctionFactory();
+        toTest = new GeoToolsFunctionFactory();
     }
 
     @Test
     public void testGetFunctionNames() {
         List<FunctionName> funcs = toTest.getFunctionNames();
-        assertEquals(1, funcs.size());
-        assertEquals("fuzzy",
-                funcs.get(0)
-                        .getName());
+        assertThat(funcs, hasSize(2));
+        assertThat(funcs.get(0).getName(), is("fuzzy"));
```

Comment:
Since the GeoToolsFunctionFactory adds the functions to an ArrayList (sequential) in the getFunctionNames call, we can assume that the retrieval order will be the same.
https://github.com/bdeining/ddf/blob/e12133920525e9610a0420da2c2a2d96fec2ac92/catalog/core/catalog-core-commons/src/main/java/ddf/catalog/impl/filter/GeoToolsFunctionFactory.java#L38